mesonbuild/compilers/c.py

```diff
@@ -710,3 +710,79 @@ def get_include_args(self, path: str, is_system: bool) -> T.List[str]:
         if path == '':
             path = '.'
         return ['--include_path=' + path]
+
+class TCCCCompiler(CCompiler):
```

Comment:
I know this is kinda pedantic, but can we call this `TinyCCmopiler`? Having 4 `C`s in a row is hard to read, and when you expand it makes no sense `TinyCCompilerCCompiler` :)